examples/inverse/plot_mixed_source_space_inverse.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Apr 20 11:03:39 2016
+
+@author: pasca
+"""
+
+import os.path as op
+import numpy as np
+import matplotlib.pyplot as plt
+import mne
+
+from mne.datasets import sample
+from mne.io import Raw
+from mne import write_source_spaces, setup_source_space
+from mne import setup_volume_source_space
+from mne import make_forward_solution
+from mne.minimum_norm import make_inverse_operator, apply_inverse_epochs
+from mne.connectivity import spectral_connectivity
+from mne.viz import circular_layout, plot_connectivity_circle
+
+# Set dir
+data_path = sample.data_path()
+sbj_id = 'sample'
+data_dir = op.join(data_path, 'MEG', sbj_id)
+sbj_dir = op.join(data_path, 'subjects')
+bem_dir = op.join(sbj_dir, sbj_id, 'bem')
+
+# Set file names
+fname_mixed_src = op.join(bem_dir, '%s-oct-6-mixed-src.fif' % sbj_id)
+fname_aseg = op.join(sbj_dir, sbj_id, 'mri/aseg.mgz')
+
+fname_model = op.join(bem_dir, '%s-5120-bem.fif' % sbj_id)
+fname_bem = op.join(bem_dir, '%s-5120-bem-sol.fif' % sbj_id)
+
+fname_raw = data_dir + '/sample_audvis_filt-0-40_raw.fif'
+fname_trans = data_dir + '/sample_audvis_raw-trans.fif'
+fname_fwd = data_dir + '/sample_audvis-meg-oct-6-mixed-fwd.fif'
+fname_cov = data_dir + '/ernoise-cov.fif'
+fname_event = data_dir + '/sample_audvis_filt-0-40_raw-eve.fif'
+
+# List of sub structures we are interested in
+'''
+labels_vol = ['Left-Accumbens-area',
+             'Left-Amygdala',
+             'Left-Caudate',
+             'Left-Hippocampus',
+             'Left-Pallidum',
+             'Left-Putamen',
+             'Left-Thalamus-Proper',
+             'Left-Cerebellum-Cortex',
+             'Brain-Stem',
+             'Right-Accumbens-area',
+             'Right-Amygdala',
+             'Right-Caudate',
+             'Right-Hippocampus',
+             'Right-Pallidum',
+             'Right-Putamen',
+             'Right-Thalamus-Proper',
+             'Right-Cerebellum-Cortex']
+'''
+# Here we select only these sub structures we want to include in the source
+# space
+labels_vol = ['Left-Amygdala',
+              'Left-Thalamus-Proper',
+              'Left-Cerebellum-Cortex',
+              'Right-Amygdala',
+              'Right-Thalamus-Proper',
+              'Right-Cerebellum-Cortex']
+
+# Setup a surface-based source space
+src = setup_source_space(sbj_id, subjects_dir=sbj_dir,
+                         spacing='oct6', add_dist=False, overwrite=True)
+
+# We create a mixed src space adding to the surface src space the volume
+# regions specified in the list labels_vol. First, read the aseg file and the
+# source space bounds using the inner skull surface
+
+# Generate the mixed source space
+for l in labels_vol:
+    # setup a volume source space of the label l
+    vol_src = setup_volume_source_space(sbj_id, mri=fname_aseg,
+                                        pos=5.0,
+                                        bem=fname_model,
+                                        volume_label=l,
```

Comment:
yes, u are right...I'm modifying the setup_volume_source_space to get as input also a list of str